Makefile

```diff
@@ -114,6 +114,9 @@ RELEASE_DIST_VER ?= $(shell hack/get_release_info.sh DIST_VER)
 RELEASE_ARCH ?= $(shell hack/get_release_info.sh ARCH)
 RELEASE_BASENAME := $(shell hack/get_release_info.sh BASENAME)
 
+# Set DOCKERPY_TEST non-empty to run a single test
```

Comment:
Comment not valid. Empty value will search whole repository for tests...

Reply:
oof, woops.  too quick to dismiss.  I see what you're saying now.

Reply:
okay, I just removed the thing.  Simpler that way.